app/errors.py

```diff
@@ -0,0 +1,19 @@
+class VaccineError(Exception):
+    ...
+    # def __str__(self):
+    #     return "All friends should be vaccinated"
+
+
+class NotVaccinatedError(VaccineError):
+    ...
+
+
+class OutdatedVaccineError(VaccineError):
+    ...
+
+
+class NotWearingMaskError(Exception):
+    masks_to_buy = 0
```

Comment:
it is a common practice to pass parameters to `__init__` in exceptions instead of making it a class attribute. Also I see what you did here - you want to track amount of masks inside of the exception but this variable would rather below to go_to_cafe function